UPGRADE-7.0.md

```diff
@@ -116,6 +116,31 @@ less files. Currently this is used to inject STATIC_CONTENT_URL which can be use
 
 For the usage in a .less file use this exact notation: url("@{STATIC_CONTENT_URL}").
 
+## CmsTextBlockBundle
+
+The text block rendering now passes all callTimeVars to the wysiwyg rendering.
+If your theme has a copy of: pkgCmsTextBlock/views/db/TPkgCmsTextBlock/standard.view.php, 
+you should add the fourth parameter to GetTextField()
+
+```php
+echo $oTextBlock->GetTextField("content", $iWidth, false, $placeholders);
+```
+
+TextBlockLookup now has 2 new methods that allow to set additional placeholder variables, that are passed to the wysiwyg text rendering.
+
+- TextBlockLookup::getRenderedText()
+- TextBlockLookup::getRenderedTextFromTextBlock()
+- TextBlockLookup::getHeadlineFromTextBlock()
+
+### Deprecations
```

Comment:
There is (already) a block "# Deprecated Code Entities" in that file.

Reply:
I moved the list of deprecated method to the list in the deprecation block